spec/views/content_assets/index.html.erb_spec.rb

```diff
@@ -2,14 +2,14 @@
 
 RSpec.describe "content_assets/index", type: :view do
   before(:each) do
-    @content_asset = FactoryBot.create(:content_asset)
-    @content_asset2 = FactoryBot.create(:content_asset)
-
-    assign(:content_assets, [@content_asset, @content_asset2])
+    @content_assets = [FactoryBot.create(:content_asset), FactoryBot.create(:content_asset)]
   end
 
   it "renders a list of content_assets" do
```

Comment:
This test might be a false positive because the `FactoryBot` objects created would both contain the same `title` and `alt` text.

Could try using the `Faker` gem to generate text and alt values for the `ContentAsset` Factory object so that they are unique or at least contain different values.

Reply:
at the moment this test is just testing for a list of (2) content assets with a title and alt text

Reply:
The rendered page will have something like this
```
<div>
  <div> Title </div>
  <div> Sample alt text </div>
</div>
<div>
  <div> Title </div>
  <div> Sample alt text </div>
</div>
```

The test is checking that the page contains the same asset `title` and `alt`.
If the test doesn't require the rendered view to contain two unique assets then the test should not loop through a list of assets.

This test will still be valid if the title and alt appeared on the rendered page 5 times. Which I don't think is the intention